django/conf/locale/es/formats.py

```diff
@@ -26,5 +26,5 @@
     "%d/%m/%y %H:%M",
 ]
 DECIMAL_SEPARATOR = ","
-THOUSAND_SEPARATOR = "."
+THOUSAND_SEPARATOR = " "  # Non-breaking space
```

Comment:
```suggestion
THOUSAND_SEPARATOR = "\xa0"  # Non-breaking space
```

Reply:
some code that uses `' ' (%20)`  instead of `\xa0`, should I change that code too? @felixxm

Reply:
> Should I change that code too?

No, it's unrelated. We shouldn't mess with formats without confirmation from native users.

Reply:
to quote a friend, the space digit separator is not standard in Spanish speaking countries except Spain.

so I modified the code. thanks @felixxm